src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -72,12 +74,55 @@ export const AddLiquidityInput: React.FC<AddLiquidityProps> = ({
 
   const { data: parsedPools } = usePools()
 
-  const foundPool = useMemo(() => {
+  const assets = useAppSelector(selectAssets)
+  const poolAssets = useMemo(() => {
+    if (!parsedPools) return []
+
+    return [...new Set(parsedPools.map(pool => assets[pool.assetId]).filter(isSome))]
+  }, [assets, parsedPools])
+
+  // TODO(gomes): Even though that's an edge case for users, and a bad practice, handling sym and asymm positions simultaneously
+  // *is* possible and *is* something that both we and TS do. We can do one better than TS here however:
+  // - When a user deposits symetrically, they can then deposit asymetrically, but only on the asset side
+  // - When a user deposits asymetrically, no matter the side, they *can* deposit symetrically on the other side
+  //   - They can also deposit asymetrically after that, but with one caveat: they can do so only if they deposited asym on the *asset* side only
+  //     In other words, if they have an active asym. RUNE position, they can't deposit symetrically after that unless they withdraw
+  //     The reason for that is that the RUNE side memo performs a nameservice operation, registering the asset address (or a placeholder)
+  //
+  //     We should handle this in the UI and block users from deposits that *will* fail, by detecting their current position(s)
+  //     and not allowing them to select the sure-to-fail deposit types
+  const defaultOpportunityId = useMemo(() => {
     if (!parsedPools) return undefined
+    if (opportunityId) return undefined
 
-    return parsedPools.find(pool => pool.opportunityId === opportunityId)
+    // since we do asset/rune/sym ordering, we can assume the the third item is sym
+    return parsedPools[2].opportunityId
```

Comment:
We can do one better than this, this works for now but is not guaranteed in the future

Reply:
cf7164938a